### ISSUE TITLE
Plotting functions for visualization

### ISSUE THREAD

**Issue by millerh1** [dmontemayor/jdrfmetabo#5]
- [ ] Heatmap showing the latent dimensions
- [ ] UMAP or tSNE 
- [ ] Scatter plot of LVs
- [ ] Other plots from the single-cell space?

**Comment by dmontemayor** [comment]
Recently I used the thermodynamic definition of entropy to visualize the sparsity of the U matrix. You can find it in the main branch. Basically, the less sparse the matrix is, the more disordered it is in terms of the weight mixing. So entropy goes down as sparsity goes up. This metric is analogous to the L1 norm used in the paper to measure sparsity.

**Comment by millerh1** [comment]
@dmontemayor Propose we need some plots that will help to assess whether we are in an overfitting regime

1. Training vs validation error -- expect them to be similar and be parallel
